content/millennium/dstu2/devices/device.md

```diff
@@ -7,10 +7,31 @@ title: Device | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+
+The Device Resource implementation is currently limited to devices implanted in a patient such as a pacemaker or insulin pump. As such, this resource currently exposes the known implant history for a patient at this site.
```

Comment:
I don't think it technically should be capitalized. It's just a normal noun, nothing proper or otherwise falling into the lovely English capitalization rules. I'll fix this instance.